### PR TITLE
Dashboard 2.0 - Community Update - Aug 23

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

Adds draft for our Dashboard 2.0 Community Update. ~There _may_ be some additional bits going in for `0.1.0` from @Steve-Mcl - so will hold off merging until I've spoken with him~.

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes

### CODE REVIEW

**Comment by joepavitt** [comment]
Checked in with Steve, the changes he has locally aren't quite ready, so we're going ahead as is

File: src/blog/2023/08/dashboard-community-update.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+I firstly need to begin with a  "Thank You" to the dozens of pre-alpha users we've had so far. Thanks for being patient whilst we're shipping fast and breaking things. We've had some great feedback, and we're working hard to implement is as best as possible.
+
+Below you'll find a summary of the changes we've made since our [last community update](/blog/2023/07/dashboard-0-1-release).
+
+## New Widgets
+
+### Template (<a href="https://flowforge.github.io/flowforge-nr-dashboard/nodes/widgets/ui-template.html" target="_blank">docs</a>)
+
+Steve has been doing some increddible work on the new `ui-template` widget. This widget allows you to create your own custom components using raw HTML, but also works with any of the components in the [Vuetify](https://vuetifyjs.com/en/components/all/) component library. It's a powerful tool that will enable users to be creative with their own widgets that are not currently available with the standard set of widgets.
```
Comment (MarianRaphael) [inline src/blog/2023/08/dashboard-community-update.md:26]:
```suggestion
Steve has been doing some incredible work on the new `ui-template` widget. This widget allows you to create your own custom components using raw HTML, but also works with any of the components in the [Vuetify](https://vuetifyjs.com/en/components/all/) component library. It's a powerful tool that will enable users to be creative with their own widgets that are not currently available with the standard set of widgets.
```

File: src/blog/2023/08/dashboard-community-update.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+![Examples of ui-template](https://flowforge.github.io/flowforge-nr-dashboard/assets/ui-template.9d278589.png)
+
+The Template node also provides access to two built-in functions that can be used to send data back to Node-RED:
+- **send(msg)**: Outputs a message (defined by the input to this function call) from this node in the Node-RED flow.
+- **submit()**: Send a `FormData` object when attached to a `<form>` element. The created object will consnist of the `name` attributes for each form element, corresponding to their respective `value` attributes.
```
Comment (MarianRaphael) [inline src/blog/2023/08/dashboard-community-update.md:32]:
```suggestion
- **submit()**: Send a `FormData` object when attached to a `<form>` element. The created object will consist of the `name` attributes for each form element, corresponding to their respective `value` attributes.
```

File: src/blog/2023/08/dashboard-community-update.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+![Examples of ui-switch](https://flowforge.github.io/flowforge-nr-dashboard/assets/ui-switch.fb5583c2.png)
+
+Adds a toggle switch to the user interface that can be rendered with a label, and traditional toggle switch, or, as in Dashboard 1.0, can be a square element with an icon & colour provided.
+
+## Fixes & Other Changes
+
+### Sidebar
+
+As requested on multiple occassions by the community when we released v0.0.4 of Dashboard 2.0, we've now added a side menu, as per Dashboard 1.0. Currently, this _just_ provides a link to the Dashboard UI, but gives us a canvas on which to expand functionality in the future.
```
Comment (MarianRaphael) [inline src/blog/2023/08/dashboard-community-update.md:44]:
```suggestion
As requested on multiple occasions by the community when we released v0.0.4 of Dashboard 2.0, we've now added a side menu, as per Dashboard 1.0. Currently, this _just_ provides a link to the Dashboard UI, but gives us a canvas on which to expand functionality in the future.
```

**Review by MarianRaphael — APPROVED** [review]

**Pull request merged**